hw3/style.css

```diff
@@ -0,0 +1,50 @@
+header{
+    background-color: red;
+    color: white;
+    font-weight: bold;
+    font-size: 30px;
+    text-align: center;
+    width: 760px;
+    margin: auto;
+    margin-bottom: 20px;
+}
+img.sunset{
+    margin-left: auto;
+    margin-right: auto;
+    display: block;
+}
+div{

```

Comment:
It is very bad practice to set styles on just tags especially div. Please add class to div and rewrite

Reply:
done